src/routes/__root.tsx

```diff
@@ -38,8 +38,8 @@ export const Route = createRootRoute({
             </div>
           </div>
         ) : (
-          <div className="footer-container w-full h-1/10">
-            <div className="mobile-nav-container w-full h-full block md:hidden lg:hidden xl:hidden 2xl:hidden">
+          <div className="footer-container w-full h-1/10 block md:hidden lg:hidden xl:hidden 2xl:hidden">
```

Comment:
Is there a specific reason why you are hiding it for every screen size? As far as I'm aware if you say `md:hidden` it will by hidden for all screen sizes `md` and bigger, similar to a media query. Just check that you need all of these.

Reply:
This is true, I realised this at some point and forgot to remove it. Thank you.